src/button/button.style.tsx

```diff
@@ -25,14 +26,21 @@ export const Main = styled.button<MainStyleProps>`
     // BUTTON STYLE + TEXT COLOR
     // -----------------------------------------------------------------------------
     ${(props) => {
+        const isDanger = props.$buttonIsDanger;
+
         switch (props.$buttonStyle) {
             case "secondary":
                 return css`
                     background-color: ${Color.Neutral[8](props)};
```

Comment:
Since it is wrapped by the `css` helper, you need not pass the `(props)` params already

Reply:
applied the suggestion